docs/topics/Gateway.md

```diff
@@ -564,6 +564,38 @@ Sent when multiple messages are deleted at once.
 | ids | array of snowflakes | the ids of the messages |
 | channel_id | snowflake | the id of the channel |
 
+### Message Reaction Add
+
+Sent when a user adds a reaction to a message.
+
+###### Message Reaction Add Event Fields
+
+| Field | Type | Description |
+|-------|------|-------------|
+| user_id | snowflake | the id of the user |
+| channel_id | snowflake | the id of the channel |
+| message_id | snowflake | the id of the message |
+| emoji | an [emoji](#DOCS_GUILD/emoji-object) object | the emoji used to react |
+
+>warn
+>The emoji object is not complete. For custom emojis, the id and name field are present. For standard emojis, only name is present, where the value is the actual emoji.
```

Comment:
Hmmm. You're correct that there is an emoji structure with only id and name, but that's the one on the channel page. The one I linked to is the emoji object on the guild page. I'll change it.